spec/commands/github/process_membership_event_spec.rb

```diff
@@ -5,48 +5,69 @@ def perform
     described_class.for(event_payload: event_payload)
   end
 
-  let!(:user) { create(:github_user, gh_id: 1) }
-  let!(:organization) { create(:organization, gh_id: 234, default_team_id: 123) }
+  let(:organization_gh_id) { 234 }
+  let(:team_id) { 123 }
+  let!(:organization) { create(:organization, gh_id: organization_gh_id, default_team_id: team_id) }
+  let(:member_payload) do
+    {
+      login: "Codertocat",
+      id: user_gh_id,
+      avatar_url: "",
+      html_url: ""
+    }
+  end
   let(:event_payload) do
-    double(
+    RecursiveOpenStruct.new(
       action: action,
-      member: double(id: 1),
-      team: double(id: 123),
-      organization: double(id: 234)
+      member: member_payload,
+      team: { id: team_id },
+      organization: { id: organization_gh_id }
     )
   end
 
-  context 'when user is added to default team in Github' do
-    let!(:organization_membership) do
-      create(
-        :organization_membership,
-        github_user: user,
-        organization: organization,
-        is_member_of_default_team: false
-      )
-    end
+  context 'when user does not exist in Froggo' do
     let(:action) { 'added' }
+    let(:user_gh_id) { 345 }
 
-    it 'user is added to default team in local DB' do
-      expect { perform }.to change { organization_membership.reload.is_member_of_default_team }
-        .from(false).to(true)
-    end
+    it { expect { perform }.to change { OrganizationMembership.count }.by(0) }
```

Comment:
Cambiaría el matcher a not_to(change...

```ruby
expect { perform }.not_to(change { OrganizationMembership.count })
```

También algo medio nitpicky pero creo que le pondría la descripción en el `it` (`it 'doesn't add a new membership' do` o algo por el estilo).